common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.h

```diff
@@ -54,10 +56,67 @@ class mon_wlan_hal_dwpal : public base_wlan_hal_dwpal, public mon_wlan_hal {
 
     // Private data-members:
 private:
+    bool dwpal_get_scan_params_fg(sScanCfgParams &params)
+    {
+        if (sizeof(params) != dwpal_nl_cmd_get(m_radio_info.iface_name,
+                                               LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS,
+                                               m_nl_buffer, NL_MAX_REPLY_BUFFSIZE)) {
+            LOG(ERROR) << "LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS failed! returned size does "
+                          "not match sScanCfgParams!";
+            return false;
+        }
+
+        if (!memcpy_s(&params, sizeof(params), &m_nl_buffer[NL_ATTR_HDR], sizeof(params))) {
```

Comment:
Pretty sure that `memcpy_s` returns 0 on success and non-zero value on error...

Reply:
So this looks like a real error - another blocker for the PR.

Reply:
fixed